### PR TITLE
[Test] - Add kfp deployment method in test post submit script

### PR DESCRIPTION
**Pull request by rui5i**
This PR adds kfp deployment method in post submit script. Current options are standalone and mkp.

Part of #2864 

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2859)

### CODE REVIEW

File: test/postsubmit-tests-with-pipeline-deployment.sh [hunk 1]
```diff
@@ -124,8 +133,14 @@ source "${DIR}/install-argo.sh"
 echo "argo installed"
 
 # Deploy the pipeline
-GCR_IMAGE_TAG=${PULL_BASE_SHA} source "${DIR}/deploy-pipeline-lite.sh"
-echo "KFP lite deployed"
+GCR_IMAGE_TAG=${PULL_BASE_SHA}
+if [ $KFP_DEPLOYMENT == standalone ]; then
+  time source "${DIR}/deploy-pipeline-lite.sh"
+  echo "KFP lite deployed"
```
Comment (numerology) [inline test/postsubmit-tests-with-pipeline-deployment.sh]:
```suggestion
  echo "KFP standalone deployed"
```

File: test/postsubmit-tests-with-pipeline-deployment.sh [hunk 2]
```diff
@@ -69,6 +74,10 @@ done
 TEST_RESULTS_GCS_DIR=gs://${TEST_RESULT_BUCKET}/${PULL_BASE_SHA}/${TEST_RESULT_FOLDER}
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
 
+if [ $KFP_DEPLOYMENT != standalone ]; then
```
Comment (numerology) [inline test/postsubmit-tests-with-pipeline-deployment.sh]: nit: always try to use curly braces for bash vars
```suggestion
if [ ${KFP_DEPLOYMENT} != standalone ]; then
```

File: test/postsubmit-tests-with-pipeline-deployment.sh [hunk 3]
```diff
@@ -124,8 +133,14 @@ source "${DIR}/install-argo.sh"
 echo "argo installed"
 
 # Deploy the pipeline
-GCR_IMAGE_TAG=${PULL_BASE_SHA} source "${DIR}/deploy-pipeline-lite.sh"
-echo "KFP lite deployed"
+GCR_IMAGE_TAG=${PULL_BASE_SHA}
+if [ $KFP_DEPLOYMENT == standalone ]; then
```
Comment (numerology) [inline test/postsubmit-tests-with-pipeline-deployment.sh]: ditto

File: test/postsubmit-tests-with-pipeline-deployment.sh [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright 2018 Google LLC
+# Copyright 2020 Google LLC
```
Comment (numerology) [inline test/postsubmit-tests-with-pipeline-deployment.sh]: Actually IMHO this should be the first date when this script is introduced, so I guess it should be 2018?

I'm not sure. Plz correct me if I'm wrong.

**Comment by rui5i** [comment]
Done. @numerology

**Comment by numerology** [comment]
/lgtm
/assign @Bobgy 

Thanks! @rui5i

**Comment by Bobgy** [comment]
LGTM

**Pull request merged**